Interfaces/About.html

```diff
@@ -0,0 +1,135 @@
+            <li><a>English</a></li>
+            <li><a>My account</a></li>
+            <li><a>logout</a></li>
+          </ul>
+        </nav>
+          <nav class="navbar">
+            <a href="../Index.html"><img class="brand-logo" src="../Images/logo_image/logo-image.png" alt="Brand-name-and-logo"></a>
+           <ul class="nav-list-items">
+            <li class="list-item"><a  href="../Index.html">Home</a></li>
+            <li class="list-item"><a  href="../Interfaces/About.html">About</a></li>
+            <li class="list-item"><a  href="#">Program</a></li>
+            <li class="list-item"><a  href="#">Sponsor</a></li>
+            <li class="list-item"><a  href="#">New</a></li>
+            <li class="list-item"><a  href="#">Ticket</a></li>
+           </ul>
+           <div class="hambuger-container ">
+           <div class="bar"></div>
+           <div class="bar"></div>
+           <div class="bar"></div>
+           </div>
+          </nav>
+        </header>
+        <section class="landing-section1">
+          <div class="intro-section1">
+            <h1 class="openingText1">
+              Quarter mboko rappers           
+            </h1>
+            
+                 <h2 class="captureText1">
+                  About Mboko gods among us 2022
+                 </h2>
+            
+  
+                 <p class="explaining-text1">
+                    Mboko drill unearths Cameroon's hidden artists. It is a place where talent is discovered as others come to do what they do best in their careers. It began in Bamenda in 2015 and has since grown to become both national and international..
+                 </p>
+  
+                 <p class="eventDate1">Please contact us via email for any further question about mboko drill 2022</p>
+                 <span class="venue1">mboko@drill.com</span>
+          </div>
+        </section>
+        
+        <section class="about-logo-container" >
+
+            <div class="text-time">
+           <h2  class="text-header2">Mboko drill gods among us 2022</h2>
+
+           <hr class="line2">
+            </div>
+
+            <div class="about-text">
+                <p>The logo of mboko drill gods among us was decided through the logo competition of from the last in 2020</p>
+            </div>
+
+            <div class="about-logo">
+                <img src="../Images/logo_image/logo-image.png">
+            </div>
+
+        </section>
+
+        <section class="recent-section">
+            <div class="text-time">
+                <h2 class="text-header2">See the past mboko drill event</h2>
+     
+                <hr class="line2">
+                 </div>
+     
+                 <div class="about-text2">
+                     <p>Take a look at the recent past two events about mboko drill gods among us.</p>
+                 </div>
+
+                 <div class="recent-carts">
+                    <div class="card card1">
+                        <h1>2020</h1>
```

Comment:
Kindly, I suggest that for accessibility your website can only contain 1 main title H1, for others use the other versions H2, H3, etc.